account_statement_import_camt/models/res_config_settings.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 CampToCamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+MODULE = __name__.split(".")[2]
```

Comment:
Seems a bit wobbly, I think you can hardcode the module name here
```suggestion
MODULE = "account_statement_import_camt"
```
There is surely a better approach but I think it's fine by doing so.